src/Types/CcdAmount.cs

```diff
@@ -136,10 +136,27 @@ public static bool TryDeserial(ReadOnlySpan<byte> bytes, out (CcdAmount? Amount,
     /// </summary>
     public byte[] ToBytes() => Serialization.ToBytes(this.Value);
 
+    /// <summary>
+    /// Return the largest of two amounts.
+    /// </summary>
+    public static CcdAmount Max(CcdAmount first, CcdAmount second)
+    {
+        if (first < second)
+        {
+            return second;
+        }
+        else
+        {
+            return first;
+        }
```

Comment:
Does this not work? (maybe you need an explicit `return` in front, not sure)
```suggestion
        first < second ? second : first
```